content/docs/setup/kubernetes/multicluster-install/vpn/index.md

```diff
@@ -7,8 +7,8 @@ aliases:
     - /docs/setup/kubernetes/multicluster-install
 ---
 
-Instructions for installing an Istio mesh across multiple clusters when pods
-in each cluster have direct network access to pods in other clusters.
+Instructions for installing an Istio [multicluster service mesh](/docs/concepts/multicluster-configuration/)
+when pods in each cluster have direct network access to pods in other clusters.
 
```

Comment:
"when pods ..." replace with 
`
where Kubernetes cluster services and applications in each Kubernetes cluster have the capability to expose their internal Kubernetes network to other Kubernetes clusters.
`